test/connected_device_operation_test.dart

```diff
@@ -2,10 +2,7 @@ import 'dart:async';
 
 import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';
 import 'package:flutter_reactive_ble/src/connected_device_operation.dart';
-import 'package:flutter_reactive_ble/src/model/characteristic_value.dart';
-import 'package:flutter_reactive_ble/src/model/qualified_characteristic.dart';
 import 'package:flutter_reactive_ble/src/model/result.dart';
-import 'package:flutter_reactive_ble/src/model/unit.dart';
```

Comment:
Prefer to import individual files instead of the library umbrella "header" (`import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';`).

Please keep the individual file imports, remove the umbrella file import, and add any missing individual file imports necessary.

This relates to every files touched in this pull request.

Reply:
Thanks for the quick review; done.